.NET/Microsoft.Recognizers.Text.NumberWithUnit/Parsers/BaseCurrencyParser.cs

```diff
@@ -123,6 +123,8 @@ private ParseResult MergeCompoundUnit(ExtractResult compoundResult)
                     };
 
                     mainUnitValue = unitValue;
+
+                    // Nothing to resolve. This happens when the entity is a currency name only (no numerical value).
```

Comment:
I don't think this is the right place for this comment. In the initial review, it was supposed to go where you had a ```return null;```